docs/pipeline.md

```diff
@@ -0,0 +1,19 @@
+# Pipeline Mock-up
+Create an infrastructure for home-labs for a very simple web app.
+
+## The objective is to gain practice on the following items:
+
+- Version Control Systems - Git, GitHub
+- CI/CD Tools - Jenkins X, Argo CD
```

Comment:
I suggest we remove Jenkins, I don't recommend learning it unless specifically required and it can be misleading for beginners. Of course this can be discussed

Reply:
Absolutely agree